checka11y.css

```diff
@@ -145,3 +145,8 @@ iframe[tabindex="-1"]::after {
 [contentEditable="true"][tabindex="-1"]::after {
   content: "ERROR: Ensure that HTML elements with editable content don't have a tab index of -1." !important;
 }
+
+u::after {
+  content: "WARNING: Using <u> -moz-element could be confused as a hyperlink, consider using a different -moz-element such as <em> or <b>" !important;
```

Comment:
Strange, this selector has 2 `content` declarations 😅 Could you rerun `npm run build`?

Reply:
@jackdomleo7 I've addressed your PR comments and reran npm run build.
I still see 2 content declarations, however on investigation I find in package.json under the `build:expanded` script if I remove `npx postcss checka11y.css --use autoprefixer --replace --no-map` I only get the 1 selector.  I'm not 100% sure what this command is doing and why it's being used so hoping you can shed some light on it

Reply:
additionally just removing the `--use autoprefixer --replace` sections also produce 1 selector but then you get the complete output of the index.html file in your terminal which is probably not wanted

Reply:
No worries, I think Autoprefixer is getting confused with something. We don't want to remove that section because we need Autoprefixer 😉